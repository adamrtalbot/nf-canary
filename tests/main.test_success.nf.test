nextflow_process {

    name "Test Process TEST_SUCCESS"
    script "main.nf"
    process "TEST_SUCCESS"

    test("Should run without failures") {

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
