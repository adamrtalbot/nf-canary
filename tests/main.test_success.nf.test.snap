{
    "Should run without failures": {
        "content": [
            {
                
            }
        ],
        "timestamp": "2023-07-20T10:50:08+0000"
    }
}